specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01/compute.json

```diff
@@ -4363,6 +4631,88 @@
       ],
       "description": "The List Availability Set operation response."
     },
+    "ProximityPlacementGroupProperties": {
+      "properties": {
+        "proximityPlacementGroupType": {
+          "type": "string",
+          "description": "Specifies the type of the proximity placement group. <br><br> Possible values are: <br><br> **Standard** <br><br> **Ultra**",
+          "enum": [
+            "Standard",
+            "Ultra"
+          ],
+          "x-ms-enum": {
+            "name": "ProximityPlacementGroupType",
+            "modelAsString": true
+          }
+        },
+        "virtualMachines": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResource"
+          },
+          "description": "A list of references to all virtual machines in the proximity placement group."
+        },
+        "virtualMachineScaleSets": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResource"
+          },
+          "description": "A list of references to all virtual machine scale sets in the proximity placement group."
+        },
+        "availabilitySets": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResource"
+          },
+          "description": "A list of references to all availability sets in the proximity placement group."
+        }
+      },
+      "description": "Describes the properties of a Proximity Placement Group."
+    },
+    "ProximityPlacementGroup": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ProximityPlacementGroupProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Specifies information about the proximity placement group."
+    },
+    "ProximityPlacementGroupUpdate": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/UpdateResource"
+        }
+      ],
+      "description": "Specifies information about the proximity placement group."
+    },
+    "ProximityPlacementGroupListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ProximityPlacementGroup"
+          },
+          "description": "The list of proximity placement groups"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of proximity placement groups. Call ListNext() with this URI to fetch the next page of proximity placement groups."
```

Comment:
can we remove the description `Call ListNext() with this URI to fetch the next page of proximity placement groups`?, not all SDKs will have method name exactly same as `ListNext()`, also this will be confusing for direct users of REST APIs since there is no such API ListNext().

Reply:
Ok , updated the description